dev/lis-gene-search-element.html

```diff
@@ -31,12 +31,15 @@ <h1>&lt;lis-gene-search-element&gt;</h1>
         This allows users to share specific pages from a search via the URL and for the search history to be navigated via the Web browser's forward and back buttons.
         If the query string parameters are present when the component loads then a search will be automatically performed with the query string parameter values.
       </p>
+      <p>
+        A search for a specific genus can be enabled by specifying the <code>genus</code> attribute in the HTML tag.
+        Not including the <code>genus</code> attribute, or including a genus that does not exist, will allow the user to select any genus.
+      </p>
       <hr>
-    </div>
 
-    <div class="uk-container uk-container-expand">
       <!-- the custom gene search element -->
-      <lis-gene-search-element id="gene-search"></lis-gene-search-element>
+      <lis-gene-search-element id="gene-search" genus="glycine"></lis-gene-search-element>
```

Comment:
Don't use this `genus` attribute by default since it's a specific use-case. Instead, mention in the text above that the search element is available in the console via the `geneSearchElement` variable and that the property can be applied to it that way.